apps/api/src/config/env-validator.ts

```diff
@@ -50,11 +50,17 @@ const validators: { [K in keyof any]: ValidatorSpec<any[K]> } = {
     default: '',
   }),
   REDIS_CACHE_SERVICE_PORT: str({
-    default: '6379',
+    default: '',
   }),
   REDIS_CACHE_SERVICE_TLS: json({
     default: undefined,
   }),
+  REDIS_CLUSTER_SERVICE_HOST: str({
+    default: '',
+  }),
+  REDIS_CLUSTER_SERVICE_PORTS: str({
+    default: '',
+  }),
```

Comment:
Not sure if we should validate the Elasticache variables as they are optional and depending on the environment deployed.